src/app.js

```
@@ -1 +1,38 @@
-// write code here
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const args = process.argv.slice(2);
+const path = require('path');
+
+function moveFile(source, destination) {
+  if (!fs.existsSync(source)) {
+    console.error('Source file does not exist');
+
+    return;
+  }
+
+  if (source !== destination) {
+    try {
+      if (
+        fs.existsSync(destination) &&
```

When the destination is interpreted as a directory (either by trailing slash or by being an existing directory), the code should throw an error if the directory does not exist. Currently, if the destination does not exist but ends with a slash, the code will attempt to move and fail with a less clear error message. Add an explicit check and error message for this case.